src/table/hover-toggle.directive.ts

```diff
@@ -0,0 +1,20 @@
+import {
+	Directive,
+	HostListener
+} from "@angular/core";
+
+@Directive({
+	selector: "[ibmHoverToggle]"
+})
+export class HoverToggle {
+	@HostListener("mouseenter", ["$event"])
+	addHoverClass(event) {
+		console.log(event.target);
+		event.target.previousElementSibling.classList.add("bx--expandable-row--hover-v2");
```

Comment:
It would be cool if class name was configurable.

Reply:
It would also be cool if `previousElementSibling` is somehow configurable. Directive name is somewhat misleading since it's not toggling a class on target.